docs_src/design/adr/devops/0010-Release-Artifacts.md

```diff
@@ -0,0 +1,95 @@
+# Release Artifacts
+
+## Status
+
+In Review 06/10/2020
+
+## Context
+
+During the Geneva release of EdgeX Foundry the DevOps WG transformed the CI/CD process with new Jenkins pipeline functionality. After this new functionality was added we also started adding release automation. This new automation is outlined in ADR 0007 Release Automation. However, in ADR 0007 only three release artifact types are outlined. This document is meant to be a living document to try to outline the artifact types for an EdgeX Foundry release and should be modified as that changes.
+
+## Release Artifact Types
+
+### Docker Images
+
+*Tied to Code Release?* Yes
+
+Docker images are released for every named release of EdgeX Foundry. During development the community releases images to the `docker.staging` repository in [Nexus](http://nexus3.edgexfoundry.org). At the time of release we promote the last tested image from `docker.staging` to `docker.release`. In addition to that we will publish the docker image on [DockerHub](https://hub.docker.com/orgs/edgexfoundry).
```

Comment:
One thing that's confusing in this ADR is that there actually are multiple project releases from EdgeX beyond the bi-annual "named" releases. Examples include:

 * SDKs
 * device services
 * application services

Some of these also result in release artifacts (i.e. app & device services) and some don't (SDKs).

Reply:
The goal is not to call out which repositories release each artifact type because that is called out in ADR 0007. Instead is just to define the artifact types and the general processes. Device and application services follow this same pattern for docker images the only difference is the schedule.

Reply:
Sure, but this section says that "Docker images are released for every named release of EdgeX Foundry."

My point is that they're also released for application and device service releases which are not "named".